reference/shaders-no-opt/asm/frag/for-loop-dedicated-merge-block-non-inverted.asm.invalid.frag

```diff
@@ -0,0 +1,11 @@
+#version 450
```

Comment:
The old non-invalid reference file lingers.

Reply:
Fixed.